web3sTests/ENS/ENSTests.swift

```diff
@@ -296,11 +298,72 @@ class ENSTests: XCTestCase {
             XCTAssertEqual(error as? EthereumNameServiceError, .ensUnknown)
         }
     }
+
+    func testGivenMainnetRegistry_WhenResolvingOnChain_ItResolvesName() async {
+        do {
+            let nameService = EthereumNameService(client: mainnetClient!)
+
+            let address = try await nameService.resolve(
+                ens: "michael-brown.eth",
+                mode: .onchain
+            )
+
+            XCTAssertEqual("0x7bcf6af56f0e4e7498b2a76d4ac8b3262ac790bb", address)
```

Comment:
My prod address is immortalised LOL 😃
Now I have to make sure I renew my ENS to that tests don't fail 😛